lib/receptor_controller/client/directive_blocking.rb

```diff
@@ -20,7 +20,7 @@ def call(body = default_body)
 
       msg_id = JSON.parse(response.body)['id']
 
-      logger.debug("Receptor response: registering message #{msg_id}, href_slug: #{body[:payload]["href_slug"]}")
+      logger.debug("Receptor response [#{ReceptorController::Client::Configuration.default.queue_persist_ref}]: registering message #{msg_id}, href_slug: #{body[:payload]["href_slug"]}")
```

Comment:
@slemrmartin Do we want to log this with every message or just once when we initialize a new worker?

Reply:
@mkanoor with every message, you can pair registration of message with response message by persist_ref